### PR TITLE
Relex feature-branches pre-live and permit hotfix changes on master

### PR DESCRIPTION
**Pull request by PhilDye**

### CODE REVIEW

File: common-flow.md [hunk 1]
```diff
@@ -56,8 +60,16 @@ interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
        release/production" state to reduce any friction with creating a new
        release.
 3. Change Branches
-    1. Each change (feature, bugfix, etc.) MUST be performed on separate
-       branches that SHOULD be referred to as "change branches".
+    1a. Before a project is Live;
```
Comment (michaelchart) [inline common-flow.md]: The 1a and 1b appear to have messed up the markdown formatting. Perhaps you need a nested list within 1.

File: common-flow.md [hunk 2]
```diff
@@ -56,8 +60,16 @@ interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
...
+       - each change (feature, bugfix, etc.) SHOULD be performed on separate 
+       branches that SHOULD be referred to as "change branches". You MAY
+       perform small (typically single-changeset) changes directly to the 
+       Master Branch, subject to not breaking other requirements.
```
Comment (michaelchart) [inline common-flow.md]: What does 

> not breaking other requirements

 mean?

**Review by michaelchart — CHANGES_REQUESTED** [review]
I like your new definitions and the spec is good, we just need to clarify the "not breaking other requirements" part I think.

File: common-flow.md (hunk 2 above)
Comment (PhilDye) [reply common-flow.md]: @michaelchart Is the local scope of 'of this spec' not implied? ie don't violate any of the other MUSTs

Comment (michaelchart) [reply common-flow.md]: Do we need to say "subject to not breaking other requirements" - just seems slightly unnecessary?

**Pull request merged**